apprentice_teacher.rb

```diff
@@ -1,22 +1,16 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'staff'
+require_relative 'Training'
+
+class ApprenticeTeacher < Staff
+  attr_reader :target_raise
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
```

Comment:
This is an attribute in more than one class, how could this bet set in the super(parent) class? Have you checked on how to use a constant to set the value of an attribute across classes but assign the value in the parent class?